### PR TITLE
Update description for TopoLVM

### PR DESCRIPTION
**Pull request by toshipp**

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Tencent Cloud File Storage](https://github.com/TencentCloud/kubernetes-csi-tencentcloud)| `com.tencent.cloud.csi.cfs` | v1.0 | A Container Storage Interface (CSI) Driver for Tencent Cloud File Storage | Persistent | Read/Write Multiple Pods | Yes |
 [Tencent Cloud Object Storage](https://github.com/TencentCloud/kubernetes-csi-tencentcloud)| `com.tencent.cloud.csi.cosfs` | v1.0 | A Container Storage Interface (CSI) Driver for Tencent Cloud Object Storage | Persistent | Read/Write Multiple Pods | No |
-[TopoLVM](https://github.com/topolvm/topolvm)| `topolvm.io` | v1.1 | A Container Storage Interface (CSI) Driver for LVM | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Expansion, Topology Aware
+[TopoLVM](https://github.com/topolvm/topolvm)| `topolvm.io` | v1.1 | A Container Storage Interface (CSI) Driver for LVM | Persistent | Read/Write Single Node | Yes | Raw Block, Expansion, Topology, Snapshot, Cloning, Storage Capacity Tracking
```
Comment (xing-yang) [inline book/src/drivers.md:119]: Do you meant RWO and RWOP?
@msau42 In this column, driver typically has either "Read/Write Single Pod" or "Read/Write Multiple Pods".  Should we change these to RWO, RWOP, RWX, etc?

Comment (msau42) [reply book/src/drivers.md:119]: Yeah I think aligning with k8s terminology makes sense

Comment (toshipp) [reply book/src/drivers.md:119]: I mean RWO and RWOP. I referred other drivers such as https://github.com/kubernetes-csi/docs/blob/4f02ba874877d285d94ca6d2bf1d08f647203130/book/src/drivers.md?plain=1#LL48C443-L48C454
By the way, I will follow the community.

Comment (msau42) [reply book/src/drivers.md:119]: If we want to realign terminology, we should do it for all the drivers.

For adding this particular entry, it looks consistent with the others, so this is fine.

**Pull request merged**